src/css/_tables.scss

```
@@ -0,0 +1,71 @@
+@use "./functions" as *;
+
+// Flat, borderless, full-width tables.
+//
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)